docs-translations/hi-IN/README.md

```diff
@@ -0,0 +1,57 @@
+# Jupyter Notebook
+
+[![Google Group](https://img.shields.io/badge/-Google%20Group-lightgrey.svg)](https://groups.google.com/forum/#!forum/jupyter)
+[![Build Status](https://travis-ci.org/jupyter/notebook.svg?branch=master)](https://travis-ci.org/jupyter/notebook)
+[![Documentation Status](https://readthedocs.org/projects/jupyter-notebook/badge/?version=latest)](https://jupyter-notebook.readthedocs.io/en/latest/?badge=latest)
+                
+अंग्रेजी संस्करण लिंक : [[English Version](http://github.com/jupyter/notebook/)]
+
+Jupyter Notebook एक इंटरैक्टिव वेब-आधारित notebook-शैली का वातावरण है।
+
+![Jupyter notebook example](resources/running_code_med.png "Jupyter notebook example")
+
```

Comment:
The [Notice block](https://github.com/jupyter/notebook#notice) appears to be missing.